solidago/pyproject.toml

```diff
@@ -18,9 +18,9 @@ classifiers = [
 ]
 keywords = ["tournesol", "collaborative recommendations", "judgement aggregation", "comparison based", "mehestan"]
 dependencies = [
-    "pandas>=1.5.3,<2.0",
+    "pandas>=1.5.3,<3.0",
```

Comment:
Not upgrading to pandas 2 like Tournesol?

Reply:
Pandas has been upgraded in Tournesol to take advantage of performance improvements. I don’t want to block the usage of Solidago in external projects that would use Pandas v1.